packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1237,6 +1237,90 @@ export class Profiles extends ProfilesCache {
         }
     }
 
+    public async basicAuthClearSecureArray(profileName?: string): Promise<void> {
+        const profAttrs = await this.getProfileFromConfig(profileName);
+        const configApi = (await this.getProfileInfo()).getTeamConfig();
+        configApi.set(`${profAttrs.profLoc.jsonLoc}.secure`, []);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.user`);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.password`);
+        await configApi.save();
+    }
+
+    public async tokenAuthClearSecureArray(profileName?: string): Promise<void> {
+        const profAttrs = await this.getProfileFromConfig(profileName);
+        const configApi = (await this.getProfileInfo()).getTeamConfig();
+        configApi.set(`${profAttrs.profLoc.jsonLoc}.secure`, []);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.tokenType`);
+        configApi.delete(`${profAttrs.profLoc.jsonLoc}.properties.tokenValue`);
+        await configApi.save();
+    }
+
+    public async handleSwitchAuthentication(node?: IZoweNodeType): Promise<void> {
+        let loginTokenType: string;
+        let serviceProfile: zowe.imperative.IProfileLoaded;
+        if (node) {
+            serviceProfile = node.getProfile();
+        } else {
+            serviceProfile = this.loadNamedProfile(node.label.toString().trim());
+        }
+        const zeInstance = ZoweExplorerApiRegister.getInstance();
+        try {
+            loginTokenType = await zeInstance.getCommonApi(serviceProfile).getTokenTypeName();
+        } catch (error) {
+            ZoweLogger.warn(error);
+            Gui.showMessage(
+                localize("handleSwitchAuthentication.tokenType.error", "Cannot switch to Token Authentication for profile {0}", serviceProfile.name)
+            );
+            return;
```

Comment:
Should this be an error or warning message instead of info? The message itself implies than an issue occurred during the switch, so I think we should increase the message severity so that users don't overlook it.